### PR TITLE
Update sitl-native-on-windows.rst

### PR DESCRIPTION
**Pull request by amilcarlucas**
* 64bit cygwin works as well
* python2 is required, python3 does not work yet

### CODE REVIEW

File: dev/source/docs/sitl-native-on-windows.rst [hunk 1]
```diff
@@ -99,7 +95,7 @@ that allow us to rebuild ArduPilot on Windows.
    +----------------+----------------------------------------------------------------------------------+
    | libxslt-devel  | Libs \| libxslt-devel: XML template library (development files)                  |
    +----------------+----------------------------------------------------------------------------------+
-   | python-devel   | Python \| python-devel: Ptyhon language interpreter                              |
+   | python-devel   | Python \| python-devel: Ptyhon 2 language interpreter                            |
```
Comment (hamishwillee) [inline dev/source/docs/sitl-native-on-windows.rst]: Typo for "Python" (was in previous version too)

**Comment by hamishwillee** [comment]
64 bit is missing one of our dependencies (procps). Any idea what has changed? @rmackay9 ?

**Comment by OXINARF** [comment]
Apparently that package is now available in Cygwin 64-bit. Also, I'm not sure if that package is still needed.

**Comment by hamishwillee** [comment]
Then as long as typo is fixed this could go in. Though would be nice to have confirmation that package is available and whether procps is a requirement.

**Comment by OXINARF** [comment]
Sorry, I should have been more clear. According to Cygwin webpage the package is now available for 64-bit.
Regarding it being a requirement or not, unfortunately the wiki only says it was for pkill, but not what required it - my guess is that it was for *sim_vehicle* script, which has now been converted to Python and tries different methods to kill processes. I'm pretty sure it isn't needed in Cygwin, but it would need to be tested by someone using it.

**Comment by hamishwillee** [comment]
Thanks for clarification. IMO this can be merged then after the typo is fixed. We should check the dependencies/instructions as a separate issue.

**Comment by amilcarlucas** [comment]
I've benn running SITL (rover and copter) on windows for the past 12 months on cygwin64, and it works

**Comment by rmackay9** [comment]
I think I'll merge this in..

**Pull request merged**